packages/goto/src/index.js

```diff
@@ -167,6 +167,8 @@ module.exports = ({
     if (abortTypes.length > 0) {
       await page.setRequestInterception(true)
       page.on('request', req => {
+        if (req.isInterceptResolutionHandled()) return
```

Comment:
use `req.isInterceptResolutionHandled ? req.isInterceptResolutionHandled() : false` for backward comat

Reply:
Same note for `req.continueRequestOverrides()` below

Reply:
If this package is consumed/used by other packages, consider using configurable priorities as suggested here https://github.com/puppeteer/puppeteer/blob/v13.0.0/docs/api.md#upgrading-to-cooperative-intercept-mode-for-package-maintainers

For easy reference, here is the pattern:

```ts
interface InterceptResolutionConfig {
  abortPriority?: number;
  continuePriority?: number;
}

// This approach supports multiple priorities based on situational
// differences. You could, for example, create a config that
// allowed separate priorities for PNG vs JPG.
const DEFAULT_CONFIG: InterceptResolutionConfig = {
  abortPriority: undefined,     // Default to Legacy Mode
  continuePriority: undefined,  // Default to Legacy Mode
};

// Defaults to undefined which preserves Legacy Mode behavior
let _config: Partial<InterceptResolutionConfig> = {};

export const setInterceptResolutionConfig = (config: InterceptResolutionConfig) =>
  (_config = { ...DEFAULT_CONFIG, ...config });

page.on('request', (interceptedRequest) => {
  if (request.isInterceptResolutionHandled?.()) return
  if (
    interceptedRequest.url().endsWith('.png') ||
    interceptedRequest.url().endsWith('.jpg')
  ) {
      interceptedRequest.abort('failed', _config.abortPriority);
    }
  else
    {
      // Here we use a custom-configured priority to allow for Opinionated
      // continuation.
      // We would only want to allow this if we had a very clear reason why
      // some use cases required Opinionated continuation.
      interceptedRequest.continue(
        interceptedRequest.continueRequestOverrides?.(),
        _config.continuePriority                        // Why would we ever want priority!==0 here?
      );
    }
});
```

Reply:
About `If this package is consumed/used by other packages` this library doesn't have too many users consuming it; the idea is to ship this as a major version, being necessary to use puppeteer v13